charts/linkerd-control-plane/values.yaml

```diff
@@ -182,6 +182,8 @@ proxy:
       request: ""
   # -- User id under which the proxy runs
   uid: 2102
+  # -- Group id under which the proxy runs
+  gid: 2102
```

Comment:
Is there an alternative solution here where we introduce a `gid` in the manifests explicitly? Your proxy-init change treats gids as optional. Being a bit more risk adverse, I'd prefer it if gids were not implicitly used here.

The only problem I can see with treating gids as optional is that it results in a bigger configuration surface area. Is there anything we can do to keep the config scope narrow _and_ allow people to explicitly use a gid? For example, setting it as `0` would work, but from a UX standpoint seeing `gid: 0` in the values.yaml file is not something I'd be a big fan of. I'm sure Helm supports optional args though in some capacity.

Reply:
We could use a special value like `-1` for this purpose and add a comment explaining it. Alternatively, we could do something like
```yaml
gid:
  customise: true # tells if the gid should be set to the custom value provided
  value: 2102
```
When I write charts I personally prefer the latter style. So unless you intervene I would implement it this way. What are your thoughts @mateiidavid?

Reply:
I decided to quickly implement it as I suggested. Just take a look and let me know what you think. Thanks!